### DISCUSSION TITLE
Starter prompt

### DISCUSSION THREAD

**Discussion by Spider201866** [FlowiseAI/Flowise#1492] [General]
Great work, recent improvements take it all to the next level.

1. Can't change Starter Prompt text size currently - I think it is defaulted to 16. So if the other text is reduced to say 12 it looks noticeably different.
2. It'd be great to be able to drag n drop Avatar images in - super quick and convenient.  But I know this could be tricky and is low priority.
3. Being able to directly in the interface adjust the Bot/User message box corners i.e. more rounded or square would be great too.
4. Altering the font type from a drop down list could be fun too. 
5. And also a drop down list for the send button logo. 
6. Radio or slide ON/OFF buttons for the Title, Welcome Message, PoweredBy would also be very helpful and great for speedy work.

**Comment by Jaredude** [comment]
These look like they belong [here](https://github.com/FlowiseAI/FlowiseChatEmbed) instead.
